pepper-apis/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -357,18 +357,21 @@ public static void start(String[] args, SendGridSupplier sendGridSupplier) {
                             }
 
                             if (hasMetPrecondition) {
+                                if (pendingEvent.getActionType() == EventActionType.ACTIVITY_INSTANCE_CREATION) {
+                                    Optional<EventConfigurationDto> eventConfOptDto =
+                                            eventDao.getEventConfigurationDtoById(pendingEvent.getEventConfigurationId());
+                                    if (!eventConfOptDto.isPresent()) {
```

Comment:
wondering if we need this eventConfOptDto.isPresent() check ? Unless manual db edits we probably will never run into it !!!